packages/stacking/src/utils.ts

```diff
@@ -261,26 +272,28 @@ function legacyHashModeToBtcAddressVersion(
 
 function _poxAddressToBtcAddress_Values(
   version: number,
-  hashBytes: Uint8Array,
+  hash: string | Uint8Array,
   network: StacksNetworkName
 ): string {
   if (!StacksNetworks.includes(network)) throw new Error('Invalid network.');
 
+  hash = typeof hash === 'string' ? hexToBytes(hash) : hash;
```

Comment:
```suggestion
  if (typeof hash === 'string') hash = hexToBytes(hash);
```

avoid the `hash = hash` situation, avoiding the re-assignation, also making the logic very slightly more readable i think

Reply:
Done ✅